setup/src/test/java/aero/minova/cas/setup/dependency/DependencyOrderTest.java

```diff
@@ -1,27 +1,103 @@
 package aero.minova.cas.setup.dependency;
 
+import org.apache.commons.io.IOUtils;
 import org.junit.jupiter.api.Test;
 
-import java.util.Arrays;
-import java.util.Scanner;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
 
 import static aero.minova.cas.setup.dependency.DependencyOrder.determineDependencyOrder;
 import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
 
-public class DependencyOrderTest {
+class DependencyOrderTest {
 	@Test
-	public void testByExample() throws Exception {
-		final var result = determineDependencyOrder(
-				new Scanner(getClass()//
-						.getClassLoader()//
-						.getResourceAsStream("dependency-graph.json"), "UTF-8")//
-						.useDelimiter("\\A")//
-						.next());
-		assertThat(result).isEqualTo(
-				Arrays.asList("aero.minova.app.i18n"//
-						, "aero.minova.cas.app"//
-						, "aero.minova.data.schema.app"//
-						, "aero.minova.contact"//
-						, "aero.minova.invoice"));
+	void determineDependencyOrderWithSimpleExampleTest() throws Exception {
+		final var result = determineDependencyOrder(retrieveContentFromResource("dependency-graph-oiltanking.swb.json"));
+		assertThat(result)
+				.containsExactly(
+						"aero.minova.app.i18n", //
+						"aero.minova.cas.app", //
+						"aero.minova.data.schema.app", //
+						"aero.minova.contact", //
+						"aero.minova.invoice"
+				);
+	}
+
+	@Test
+	void determineDependencyOrderWithComplexExampleTest() throws Exception {
+		final var result = determineDependencyOrder(retrieveContentFromResource("dependency-graph-skyber.awb.app.json"));
```

Comment:
skyber -> skyfall? or something

Reply:
Erledigt. Sqash-and-Merge Richtung `main` **nicht** vergessen.